ui/app/webpack.dev.ts

```diff
@@ -25,22 +22,6 @@ declare module 'webpack' {
   }
 }
 
-// Get dev server HTTPS options
-function getHttpsConfig() {
-  // If key/cert not specified, just use the default self-signed cert
-  if (
-    process.env.SSL_KEY_FILE === undefined ||
-    process.env.SSL_CRT_FILE === undefined
-  ) {
-    return true;
-  }
-
-  return {
-    key: fs.readFileSync(process.env.SSL_KEY_FILE),
-    cert: fs.readFileSync(process.env.SSL_CRT_FILE),
-  };
-}
-
```

Comment:
Don't think we should remove this either. As someone who does a lot of `localhost` dev across different codebases/projects, adding a common SSL cert to my local machine's keychain to stop Chrome from complaining is something I definitely use. It really comes in handy if you have some project that you do local dev on that adds an HSTS to Chrome for localhost (which means Chrome always redirects to HTTPS for localhost).

Reply:
I remove it because it's bothering me and I don't have your setup and I'm pretty sure planty of user won't have it. We can keep it if you have a way to handle it with no particular setup. For example, with a condition that would enable it if you have the correct environment.